### PR TITLE
Update and release Tidal Offline

### PR DESCRIPTION
**Pull request by SamDesmondKing**
Updated tidal-offline.html with feedback gathered [here](https://docs.google.com/document/d/1f9L3SmIdojKbidGqhawPZUV9xXh2CUeE6kNANTotXbU/edit), and added this page to the main sidebar for release.

It's under `Introduction` in this PR. I initially assumed it should go under `Discovery`, but it contains information on processes outside the discovery phase (like source code analysis - which is `Assess`). This will also allow the page to grow and include any Tidal process which can be completed offline, rather than just discovery.  However this is just a thought - if you think it should go under `Discovery` instead please let me know.

### CODE REVIEW

File: pages/tidal tools/tidal_offline.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ First, install Docker on both your internet-connected and offline server, and en
 
 Follow [this guide](https://guides.tidalmg.com/tidal-tools.html) to install Tidal Tools on your internet connected server.
 
+Next, install Tidal tools on your offline server with one of two options:
```
Comment (prees1) [inline pages/tidal tools/tidal_offline.md:22]:
```suggestion
Next, install Tidal Tools on your offline server with one of two options:
```

**Review by prees1 — COMMENTED** [review]
👍 looks great Sam. I think good to ship.

I think your decision on the menu location is a good one. 👍

**Review by jmariomejiap — APPROVED** [review]
Looks really nice 🚀

**Comment by SamDesmondKing** [comment]
Thanks guys, I'll go ahead and merge.

**Pull request merged**